src/main/java/uk/gov/hmcts/ccd/domain/service/aggregated/AuthorisedGetCaseViewOperation.java

```diff
@@ -52,6 +53,17 @@ public CaseView execute(String jurisdictionId, String caseTypeId, String caseRef
         return filterUpsertAccess(caseType, userRoles, caseView);
     }
 
+    @Override
+    public CaseHistoryView execute(String jurisdictionId, String caseTypeId, String caseReference, Long eventId) {
```

Comment:
now the AuthorisedGetCaseViewOperation has got to execute methods returning different types. I find it a bit confusing. Shall we rename them to 
getCaseView and getHistoryView?

Reply:
actually I wonder whether it would be better to introduce a completely new operation GetCaseHistoryViewOperation with it's own execute. I think that would be better

Reply:
Yeah will refactor the code as get case view and case history are two separate use cases(operations).

Reply:
Done, created new GetCaseHistoryViewOperation